--- efl-1.26.0/src/lib/evas/meson.build~	2021-12-26 11:44:23.000000000 +0000
+++ efl-1.26.0/src/lib/evas/meson.build	2021-12-28 16:24:07.821226637 +0000
@@ -11,7 +11,6 @@
 webpdemux = dependency('libwebpdemux', version: ['>=0.5.0'], required: get_option('evas-loaders-disabler').contains('webp') == false)
 libopenjp2 = dependency('libopenjp2', required: get_option('evas-loaders-disabler').contains('jp2k') == false)
 libavif = dependency('libavif', required: get_option('evas-loaders-disabler').contains('avif') == false, version: '>= 0.8.2')
-heif= dependency('libheif', required: get_option('evas-loaders-disabler').contains('heif') == false)
 
 evas_image_loaders_file = [
      ['avif',    'shared', [libavif]],
@@ -19,7 +18,6 @@
      ['eet',     'static', [eet]],
      ['generic', 'shared', [rt]],
      ['gif',     'shared', [giflib]],
-     ['heif',    'shared', [heif]],
      ['ico',     'shared', []],
      ['jpeg',    'static', [jpeg]],
      ['jp2k',    'shared', [libopenjp2]],
--- efl-1.26.0/src/lib/evas/meson.build~	2021-12-28 16:24:07.000000000 +0000
+++ efl-1.26.0/src/lib/evas/meson.build	2021-12-28 16:33:15.903219006 +0000
@@ -6,7 +6,6 @@
 
 png = dependency('libpng')
 tiff = dependency('libtiff-4', required: get_option('evas-loaders-disabler').contains('tiff') == false)
-giflib = cc.find_library('gif')
 webp = dependency('libwebp', version: ['>=0.5.0'], required: get_option('evas-loaders-disabler').contains('webp') == false)
 webpdemux = dependency('libwebpdemux', version: ['>=0.5.0'], required: get_option('evas-loaders-disabler').contains('webp') == false)
 libopenjp2 = dependency('libopenjp2', required: get_option('evas-loaders-disabler').contains('jp2k') == false)
@@ -17,7 +16,6 @@
      ['bmp',     'shared', []],
      ['eet',     'static', [eet]],
      ['generic', 'shared', [rt]],
-     ['gif',     'shared', [giflib]],
      ['ico',     'shared', []],
      ['jpeg',    'static', [jpeg]],
      ['jp2k',    'shared', [libopenjp2]],
